lib/enum/arrays.rb

```diff
@@ -1,21 +1,29 @@
+require 'pry'
 class Array
 
   def sum_of_numeric_elements
+    reduce(0) {|sum, element| (element.class == Fixnum || element.class == Float)? sum += element : sum}
```

Comment:
You can do this a lot simpler. All we need is to test whether an element is Numeric, and then sum those elements. We can do it like this: 

`select{ |element| element.is_a? Numeric }.reduce(:+)`